documentation/cicdgen.asciidoc

```diff
@@ -0,0 +1,19 @@
+:toc:
+toc::[]
+
+= cicdgen
+
+The `cicdgen` commandlet allows to install and setup https://github.com/devonfw/cicdgen[cicdgen].
+The arguments (`devon cicdgen «args»`) are explained by the following table:
+
+.Usage of `devon cicdgen`
+[options="header"]
+|=======================
+|*Argument(s)*   |*Meaning*
+|`setup`                 |setup cicdgen (install and verify)
+|`update`                |update cicdgen (reinstall with @latest version and verify)
+|`java «args»`           |generate cicd files for the currect devon4java project
+|`ng «args»`             |generate cicd files for the currect devon4ng project
+|`node «args»`           |generate cicd files for the currect devon4node project
```

Comment:
I think @maybeec expected this to be integrated in the according commands.
IMHO to do that easily, we can build a bridge in those commandlets.
So 
* `devon java cicd «args»` will be an alias/synonym that delegates to `devon cicdgen java «args»`.
* `devon ng cicd «args»` will be an alias/synonym that delegates to `devon cicdgen ng «args»`.
* `devon node cicd «args»` will be an alias/synonym that delegates to `devon cicdgen node «args»`.

If you have the chance to do that today or on Monday, it would be awesome. Delegation is simply done by `doDevonCommand` function. You will find several examples in existing commandles (esp. `build` is full of those).

Reply:
This functionality will be added in the next update of this PR